packages/info-box/InfoBox.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { cnb } from "cnbuilder";
+
+// import specific templates for the component
+import { XssSafeContent } from "@squiz/xaccel-xss-safe-content";
+import { Container } from "../grids/Container";
+import { LinkButton } from "../links/LinkButton";
+
+/**
+ * A container with a gray background that includes a
+ * title and text content with optional image and CTA button.
+ *
+ * @param {string} containerClassName
+ * Use this to apply classes to the box's outer container.
+ *
+ * @param {string} title
+ * A heading displayed at the top of the container
+ *
+ * @param {string} content
+ * WYSIWYG text content
+ *
+ * @param {object} imageData
+ * The image's data object of the selected Matrix asset. We need the URL and alt text.
+ *
+ * @param {string} caption
+ * A caption to pair with the selected image.
+ *
+ * @param {string} credit
+ * A credit to display with the image caption.
+ *
+ * @param {string} imagePlacement
+ * The selected image can display above or below the content.
+ *
+ * @param {string} buttonText
+ * Text on the button.
+ *
+ * @param {string} externalUrl
+ * An external page for the button to link to.
+ *
+ * @param {string} internalLinkUrl
+ * The url to another page asset in Matrix.
+ *
+ * @param {boolean} isNewWindow
+ * Choose if the button link should open in a new window.
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export default function InfoBox({
+  containerClassName,
+  title,
+  content,
+  imageData,
+  caption,
+  credit,
+  imagePlacement,
+  buttonText,
+  externalUrl,
+  internalLinkUrl,
+  isNewWindow,
+}) {
+  const captionCredit =
+    caption && credit ? `${caption} | ${credit}` : caption || credit;
+  const hasImage = imageData?.url;
```

Comment:
@iamrentman I pulled in your branch, and updated the InfoBox component to use this check for whether an image exist instead of checking on `image` directly. This seems to work better for me when I tested my component. Let me know if that works for you.